src/main/resources/log4j2.properties

```diff
@@ -4,17 +4,14 @@ appenders = console, file
 
 appender.console.type = Console
 appender.console.name = STDOUT
-appender.console.layout.type = JSONLayout
-appender.console.layout.properties = true
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : "default"}%n
 
 appender.file.type = File
```

Comment:
Please, comment section between line 10 to 17 because this logs are not necessary